game/video_driver.cc

```diff
@@ -77,6 +70,12 @@ Window::Window(unsigned int width, unsigned int height, bool fs): m_windowW(widt
 		SDL_GL_CreateContext(screen);
 
 	}
+// 	{
+// 		SDL_Surface* icon = SDL_LoadBMP(findFile("icon.bmp").string().c_str());
+// 		SDL_SetWindowIcon(screen,icon);
```

Comment:
why comments? why don't we wrap it in #ifdefs?

Reply:
Even better if there's no need for it anymore: delete it